examples/azure/10-ark-config.yaml

```diff
@@ -24,9 +24,7 @@ persistentVolumeProvider:
     apiTimeout: <YOUR_TIMEOUT>
 backupStorageProvider:
   bucket: <YOUR_BUCKET>
-  azure:
-    location: <YOUR_LOCATION>
-    apiTimeout: <YOUR_TIMEOUT>
```

Comment:
The slow API call issue is only with snapshot operations -- I wasn't actually using the timeout value for object storage API calls before anyway. I could keep it and switch to actually using it in the code, but I don't think it's necessary.